.travis.yml

```diff
@@ -2,10 +2,10 @@ language: node_js
 
 cache:
   directories:
-  - node_modules
+    - node_modules
 
 node_js:
-  - '8.9.2'
+  - 8.9.2
```

Comment:
The current LTS version is 8.9.4. Should we update this to match?

Reply:
Probably needs npm etc upgrading too. Would rather update and test that as a separate ticket.

Reply:
..done it anyway - guess the build will tell us what we need to know!